compiler/rustc_mir_transform/src/lower_intrinsics.rs

```diff
@@ -149,6 +149,29 @@ impl<'tcx> MirPass<'tcx> for LowerIntrinsics {
                             terminator.kind = TerminatorKind::Goto { target };
                         }
                     }
+                    sym::read_via_copy => {
+                        if args.len() != 1 {
+                            span_bug!(terminator.source_info.span, "Wrong number of arguments");
+                        };
+                        let arg = args.pop().unwrap();
```

Comment:
`let Some(arg) = args.get() else { ... }` maybe?

Reply:
Oh, I missed that this wasn't actually equivalent. Feel free to ignore wrong things like this next time

Reply:
No worries; was a good excuse to use #76310 🙃